package.json

```diff
@@ -14,10 +14,10 @@
     "rollup-plugin-terser": "^7.0.2"
   },
   "dependencies": {
-    "@duckduckgo/autoconsent": "^3.0.4",
+    "@duckduckgo/autoconsent": "^4.1.0",
     "@duckduckgo/autofill": "github:duckduckgo/duckduckgo-autofill#6.1.1",
     "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#3.4.1",
-    "@duckduckgo/privacy-dashboard": "github:duckduckgo/privacy-dashboard#1.3.0",
+    "@duckduckgo/privacy-dashboard": "github:muodov/privacy-dashboard#e6b0b332ca8cad05bd0b57f5bd886ad21e91c1f3",
```

Comment:
Should be pointing to the ddg repo

Reply:
PD is released, I've pinned it to the production version